README.md

```diff
@@ -7,11 +7,46 @@ Every pipeline requires https://github.com/openvinotoolkit/openvino for C++ to b
 
 Build the pipelines and `user_ov_extensions`
 
+1.Clone submodules:
 ```sh
+git clone https://github.com/wenyi5608/openvino.genai.git -b wenyi5608-chatglm
+cd openvino.genai
 git submodule update --init
-mkdir ./build/ && cd ./build/
+```
+
+2.Build & Run
+
+For Linux
+
+Compile the project using CMake:
+```sh
 source <OpenVINO dir>/setupvars.sh
-cmake -DCMAKE_BUILD_TYPE=Release ../ && cmake --build ./ --config Release -j
+cmake -B build
+cmake --build build -j --config Release
 ```
+Usage: `chatglm <openvino_model.xml> <tokenizer.xml> <detokenizer.xml> "device"`
+
+Example: `./build/llm/chatglm_cpp/chatglm ./chatglm3-6b/openvino_model.xml ./tokenizer.xml ./detokenizer.xml "CPU"`
+
+For Windows
+
+Compile the project using CMake:
+```sh
+<OpenVINO dir>\setupvars.bat
+cmake -B build
+cmake --build build -j --config Release
+```
+
+copy user_ov_extensions.dll and its dependent dll to chatglm.exe directory
+
+```sh
+$copy build\thirdparty\openvino_contrib\modules\custom_operations\user_ie_extensions\Release\user_ov_extensions.dll build\llm\chatglm_cpp\Release\
+$copy build\_deps\fast_tokenizer-src\third_party\lib\icudt70.dll build\llm\chatglm_cpp\Release\
+$copy build\_deps\fast_tokenizer-src\third_party\lib\icuuc70.dll build\llm\chatglm_cpp\Release\
+$copy build\_deps\fast_tokenizer-src\lib\core_tokenizers.dll build\llm\chatglm_cpp\Release\
+```
+Usage: `chatglm.exe <openvino_model.xml> <tokenizer.xml> <detokenizer.xml> "device"`
+
+Example: `build\llm\chatglm_cpp\Release\chatglm.exe .\chatglm3-6b\openvino_model.xml .\tokenizer.xml .\detokenizer.xml "CPU"`
```

Comment:
I suppose chatglm specific things must be in llm/chatglm_cpp/README.md only

Reply:
I update the README.md, and put the chatglm instruction guide in the chatglm directory.